x/finality/keeper/power_dist_change.go

```diff
@@ -126,15 +126,14 @@ func (k Keeper) HandleFPStateUpdates(ctx context.Context, prevDc, newDc *ftypes.
 	}
 }
 
-// handleActivatedFinalityProvider updates the signing info start height or create a new signing info
-func (k Keeper) handleActivatedFinalityProvider(ctx context.Context, fpPk *bbn.BIP340PubKey) error {
+// HandleActivatedFinalityProvider updates the signing info start height or create a new signing info
+func (k Keeper) HandleActivatedFinalityProvider(ctx context.Context, fpPk *bbn.BIP340PubKey) error {
 	signingInfo, err := k.FinalityProviderSigningTracker.Get(ctx, fpPk.MustMarshal())
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
 	if err == nil {
 		// reset signing info
 		signingInfo.StartHeight = sdkCtx.HeaderInfo().Height
 		signingInfo.JailedUntil = time.Unix(0, 0)
-		signingInfo.MissedBlocksCounter = 0
```

Comment:
This line is the meat of this PR to align with cosmos' jailing mechanism https://github.com/cosmos/cosmos-sdk/blob/c452721442546da82d199da6d116405819f1b400/x/slashing/keeper/hooks.go#L32, which does not reset missed block counter if the signing info already exists